packages/cms/scripts/images-resize.ts

```diff
@@ -0,0 +1,76 @@
+const sharp = require('sharp');
+const fs = require('fs');
+const path = require('path');
+
+import { sizes } from '@corona-dashboard/common/src/config';
```

Comment:
If config is something to be imported from common, then the common package should export it. It's not ok for one package to directly import source files from another package. The public facing files of common live in `dist` and are exposed via the package definitions. 

You probably just need to add the config to the exports list in src/index.ts

```suggestion
import { sizes } from '@corona-dashboard/common';
```